src/index.ts

```diff
@@ -1,6 +1,6 @@
 // main symbols
 export { SearchCore } from './SearchCore';
 export { provideCore } from './provideCore';
-export { SandboxEndpoints } from './constants';
+export { provideEndpoints, SandboxEndpoints } from './constants';
```

Comment:
Can you export Environment and CloudRegion from here as well?

Reply:
Ah, thought I did it. Now exported